src/index.html

```diff
@@ -2,16 +2,137 @@
 <html lang="en">
   <head>
     <meta charset="UTF-8">
-    <meta
-      name="viewport"
+    <meta name="viewport"
       content="width=device-width, user-scalable=no, initial-scale=1.0, maximum-scale=1.0, minimum-scale=1.0"
     >
     <meta http-equiv="X-UA-Compatible" content="ie=edge">
     <title>HTML Form</title>
     <link rel="stylesheet" href="./style.css">
   </head>
   <body>
-    <h1>HTML Form</h1>
+    <form action="https://mate-academy-form-lesson.herokuapp.com/create-application" method="post">
+      <fieldset class="group">
+        <legend>Personal information</legend>
+        <label class="label">
+          Surname:
+          <input
+            type="text"
+            name="surname"
+            required
+            minlength="2"
+            autocomplete="off"
+          >
+        </label>
+        <label class="label">
+          Name:
+          <input
+            type="text"
+            name="name"
+            required
+            minlength="2"
+            autocomplete="off"
+          >
+        </label>
+        <label class="label">
+          How old are You?
+          <input
+            type="number"
+            name="age"
+            value="12"
+            min="1"
+            max="100"
+          >
+        </label>
+        <label class="label">
+          Full date of birth:
+          <input type="date" name="date-of-birth">
+        </label>
+        <label class="label">
+          I accept the term of the agreement
+          <input
+            type="checkbox"
+            name="term-and-agreement"
+            required
+          >
+        </label>
+      </fieldset>
+
+      <fieldset class="group">
+        <legend>Registration</legend>
+        <label class="label">
+          E-mail:
+          <input
+            type="email"
+            name="e-mail"
+            required
+            placeholder="email@example.com"
+          >
+        </label>
+        <label class="label">
+          Password:
+          <input
+            type="password"
+            name="password"
+            required
+            maxlength="10"
+          >
+        </label>
+      </fieldset>
+
+      <fieldset class="group">
+        <legend>An interesting fact about you!</legend>
+        <label class="label">
+          Do you love cats?
+          <input
+            type="radio"
+            value="yes"
+            name="love-cats"
+          > Yes
+          <input
+            type="radio"
+            value="no"
+            name="love-cats"
+          > No
+        </label>
```

Comment:
each input should be wrapped into a label here
```suggestion
        <div class="label">
          Do you love cats?
          <label>
            <input
              type="radio"
              value="yes"
              name="love-cats"
            > 
            Yes
          </label>
         
          <label>
            <input
              type="radio"
              value="no"
              name="love-cats"
            > 
            No
          </label>
```